### ISSUE TITLE
Backend Admin-Module Access blocked when BE-LDAP Login is enabled.

### ISSUE THREAD

**Issue by Prospero256** [xperseguers/t3ext-ig_ldap_sso_auth#204]
We recently installed version 3.8.0 of the extension.

We have local Admins, that are able to log into the BE without problems (backend.BEfailsafe is set to 1).

But when the parameter "enableBELDAPAuthentication" is set to 1, it is no longer possible to access the admin modules within the backend, that require an additional login (maintenance, settings). It only says something along the lines of "password unknown".  All we can do then is to manually edit the LocalConfiguration.php, to reset it to 0. 

The login into the backend itself seems unaffected by this bug.

**Comment by saifulislam82** [comment]
Hi,

I have enabled local Backend users to get access without ldap specially for local admin user. I did that from "Admin Tools > Settings > Extension Configuration > ig_ldap_sso_auth > Backend. I ticked that option Backend users to get access. 

From backend I saw apart from enableBELDAPAuthentication and enableFELDAPAuthentication, 'BEfailsafe' and 'throwExceptionAtLogin'  are set to 1

But now I can login with local admin user but I can't access to settings. Password says incorrect. 

I am using typo3 11.5.36 with PHP 8.2.  

I can't login using ldap user and also, I see empty Import LDAP users (Frontend) and Backend. 

Did you able to login with ldap user? Which PHP version are you using?
